pkg/options/clean.go

```diff
@@ -31,10 +31,14 @@ type Clean struct {
 	Logs      bool `json:"logs,omitempty"`
 }
 
-// NewCleanOptions instantiate a new Clean from the cleanString
+// NewCleanOptions instantiate a new Clean from the cleanString and keepString
 // The clean string is lowercase clean options comma separated like: etcd,binaries ...
-func NewCleanOptions(cleanString string) *Clean {
-	return newOptions(cleanString, &Clean{}).(*Clean)
+// keepString takes precedence over the clean one
```

Comment:
Can we add something like `keepString takes precedence over the clean one` in the docs for the `keep` command?

Reply:
addressed